### DISCUSSION TITLE
How To Install and Use Specky

### DISCUSSION THREAD

**Discussion by Fartedbutpooped27** [kitzeslab/opensoundscape#677] [Q&A]
Hi, 

I am following the instructions on how to use Specky at https://github.com/rhine3/specky. 

However, when running: 

conda env create --file conda_environment.yaml --name specky_env

I get: 

SpecNotFound: Invalid name 'specky_env', try the format: user/package

Also, where exactly is conda_environment.yaml?  Is that a file? I don't see it within the Github

**Comment by rhine3** [comment]
Hi there, looks like those installation instructions are out of date for your current version of `conda` and needed to be updated with instructions about which YAML name to use. I updated them; hope this helps. 

For any future questions about Specky please check out the Specky Discussions page: https://github.com/rhine3/specky/discussions/15

However instead of Specky our lab now uses Jupyter Notebooks similar to the following: https://github.com/kitzeslab/bioacoustics-cookbook/blob/main/top-down-listening.ipynb
